code/datums/status_effects/buffs.dm

```diff
@@ -450,3 +450,77 @@
+	owner.add_trait(TRAIT_PACIFISM, "hippocraticOath")
+	var/datum/atom_hud/H = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	H.add_hud_to(owner)
+	return ..()
+
+/datum/status_effect/hippocraticOath/on_remove()
+	owner.remove_trait(TRAIT_PACIFISM, "hippocraticOath")
+	var/datum/atom_hud/H = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	H.remove_hud_from(owner)
+
+/datum/status_effect/hippocraticOath/tick()
+	//Checks to make sure the rod is still in their hand
+	if(iscarbon(owner))
+		var/mob/living/carbon/itemUser = owner
+		var/obj/item/heldItem = itemUser.get_item_for_held_index(hand)
+		if(heldItem == null || heldItem.type != /obj/item/rod_of_asclepius)
+			var/obj/item/rod_of_asclepius/newRod = new /obj/item/rod_of_asclepius(itemUser.loc)
+			newRod.flags_1 = NODROP_1 | DROPDEL_1
+			newRod.desc = "A short wooden rod with a mystical snake inseparably gripping itself and the rod to your forearm. It flows with a healing energy that disperses amongst yourself and those around you. "
+			newRod.icon_state = "asclepius_active"
+			newRod.activated = TRUE
+			if(!itemUser.has_hand_for_held_index(hand))
+				//If user does not have the corresponding hand anymore, give them one and return the rod to their hand
+				if(((hand % 2) == 0))
+					var/obj/item/bodypart/L = itemUser.newBodyPart("r_arm", FALSE, FALSE)
+					L.attach_limb(itemUser)
+					itemUser.put_in_r_hand(newRod)
+				else
+					var/obj/item/bodypart/L = itemUser.newBodyPart("l_arm", FALSE, FALSE)
+					L.attach_limb(itemUser)
+					itemUser.put_in_l_hand(newRod)
+				to_chat(itemUser, "<span class='notice'>Your arm suddenly grows back with the Rod of Asclepius still attached!</span>")
+			else
+				//Otherwise get rid of whatever else is in their hand and return the rod to said hand
+				itemUser.dropItemToGround(itemUser.get_item_for_held_index(hand))
+				if(((hand % 2) == 0))
+					itemUser.put_in_r_hand(newRod)
+				else
+					itemUser.put_in_l_hand(newRod)
+				to_chat(itemUser, "<span class='notice'>The Rod of Asclepius suddenly grows back out of your arm!</span>")
+		//Because a servant of medicines stops at nothing to help others, lets keep them on their toes and give them an additional boost.
+		if(itemUser.health < itemUser.maxHealth)
+			new /obj/effect/temp_visual/heal(get_turf(itemUser), "#375637")
+		itemUser.adjustBruteLoss(-1.5)
+		itemUser.adjustFireLoss(-1.5)
+		itemUser.adjustToxLoss(-1.5, forced = TRUE) //Because Slime People are people too
+		itemUser.adjustOxyLoss(-1.5)
+		itemUser.adjustStaminaLoss(-1.5)
+		itemUser.adjustBrainLoss(-1.5)
+		itemUser.adjustCloneLoss(-1.5)
+	//Heal all those around you, unbiased
+	for(var/mob/living/L in view(7, owner))
+		if(L.health < L.maxHealth)
+			new /obj/effect/temp_visual/heal(get_turf(L), "#375637")
+		L.adjustBruteLoss(-3.5)
+		L.adjustFireLoss(-3.5)
+		L.adjustToxLoss(-3.5, forced = TRUE) //Because Slime People are people too
+		L.adjustOxyLoss(-3.5)
+		L.adjustStaminaLoss(-3.5)
+		L.adjustBrainLoss(-3.5)
+		L.adjustCloneLoss(-3.5)
```

Comment:
This heals simples by 21 HP per tick, which is every second or every 2 seconds for status effects? Completely removes medbay from the game

Reply:
Both the 1.5 and 3.5 values are too high imo.

Also see nasku's cloning damage comment.

Reply:
*up to 18.5 hp a tick, in very rare cases, where ticks are 2.5 seconds each. I can agree on removing cloneloss, and maybe lowering the regen to about 2/2.5 ler tick though.

Reply:
Wait can someone explain how this would heal simples for more than carbons?

Reply:
They have "damage" not brute,burn,etc.

Reply:
Yes, and its stored as Brute. So they'll only accept the adjustBrute and ignore the rest. I've tested this.

Reply:
If that's not the case i'm not sure how or even when anyone mentioned it would heal simples more than carbons.

Reply:
@TheDreamweaver exactly how did you test it? I'm looking at the damage procs right now all of them just adjust health so you're incorrect.

https://github.com/tgstation/tgstation/blob/5a08a3aad0e1684fb501eb6e4f7be54311d66e0a/code/modules/mob/living/simple_animal/damage_procs.dm

Reply:
They default to 1 for all damage but stamina, it's just not noticeable because barely any traditional healing interacts with simple mobs. This regeneration would be vastly more effective on simple mobs, yeah.

Reply:
Sorry, I was misinformed by another code then. I assumed since adjustBrute did work correct, the rest held true too. I actually originally had a separate check for simples, but I was told to combine it as it "wouldn't matter"

Reply:
i thought so as well before checking, the misconception comes from the fact that only brute packs interact with mobs when trying to heal them

Reply:
Also as a reminder, I have tick interval set to 25 which means that tick() only gets called every 2.5 seconds so it should only heal others at a little above 1 damage per second, and the owner itself at 2 damage per second.